internal/config/config_test.go

```diff
@@ -38,6 +39,42 @@ func TestParse(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "valid env",
+			file: "testdata/valid_env.yaml",
+			want: Config{
+				LavaVersion: "v1.0.0",
+				ChecktypeURLs: []string{
+					"mychecktypes.json",
+				},
+				Targets: []Target{
+					{
+						Identifier: "example.com",
+						AssetType:  types.DomainName,
+					},
+				},
+			},
+			envs: map[string]string{
+				"TARGET":           "example.com",
+				"CHECK_types_URL1": "mychecktypes.json",
```

Comment:
I would just use `checktypes.json`.

```suggestion
				"CHECK_types_URL1": "checktypes.json",
```